packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -3899,54 +3957,38 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   moveFrom(resourceId) {
+    // On moveFrom ask the developer to re capture it moveFrom  then updateMedia
```

Comment:
"re capture it moveFrom then updateMedia" - this comment could be improved... (there is a double space, it's not grammatically correct etc.) also I don't know what the "re capture it" part means,

Reply:
will handle this in the next pr for documentation